pkg/sql/codegen/xgboost/template.go

```diff
@@ -53,10 +54,11 @@ def xgb_dataset(fn, dataset_sql):
     return xgb.DMatrix(fn)
 
 dtrain = xgb_dataset('train.txt', '''{{.TrainSelect}}''')
-dtest = xgb_dataset('test.txt', '''{{.ValidationSelect}}''')
+# FIXME(weiguoz): bring dtest back when VALIDATE clause is ready
+# dtest = xgb_dataset('test.txt', '''{{.ValidationSelect}}''')
 
 bst = xgb.train(model_params, dtrain, **train_params)
-bst.save_model("my_model")
+bst.save_model('''{{.Save}}''')
```

Comment:
Thought the model is saved by `executor.go` by tar all the working directory, so this model save directory can be hard coded maybe?

Reply:
Good idea